src/library/Uno.Material/Styles/Application/Converters.xaml

```diff
@@ -44,4 +44,7 @@
 															NotEmptyOrNullValue="Visible"
 															EmptyOrNullValue="Collapsed" />
 
+	<converters:EnumerableConverter x:Key="Material_EnumerableStringJoin"
```

Comment:
Underscore in the resource name was dropped in #392. Moreover, this is an unused resource.

Reply:
Oh, I wrote it. Prior to #392... 😬

Reply:
yeah I don't think we actually need this anymore. I'm going to remove it

Reply:
removed